Src/BootCamp.Chapter/Program.cs

```diff
@@ -1,12 +1,108 @@
 ﻿using System;
+using System.Globalization; // NumberStyles.Float, CultureInfo.InvariantCulture
 
 namespace BootCamp.Chapter
 {
     class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            CheckBMI("Tom", "Jefferson", 19, 50, 156.5f);
+            CheckBMI();
+        }
+
+        static void CheckBMI()
+        {
+            string userName;
+            CheckForUserInput("Please input your name.", out userName);
+
+            string userSurname;
+            CheckForUserInput("Please input your surname.", out userSurname);
+
+            Console.WriteLine("Hi " + userName + " " + userSurname + "!");
+
+            int userAge;
+            CheckForUserInput("Please input your age.", out userAge);
+
+            float userWeight;
+            CheckForUserInput("Please input your weight in kg.", out userWeight);
+            float userHeight;
+            CheckForUserInput("Please input your height in cm.", out userHeight);
+
+            CalculateBMI(userName, userSurname, userAge, userWeight, userHeight);
+        }
+
+        static void CheckBMI(in string userName, in string userSurname, in int userAge, in float userWeight, in float userHeight)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Rename method 'CheckBMI' to match pascal case naming rules, consider using 'CheckBmi'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4860292)